pantheon/src/main/java/tech/pegasys/pantheon/controller/MainnetPantheonController.java

```diff
@@ -206,6 +206,12 @@ private MainnetPantheonController(
           } catch (final IOException e) {
             LOG.error("Failed to close storage provider", e);
           }
+
```

Comment:
We should stop the synchronizer prior to the storage.  The shutdown process could try to write more data.